include/variables.h

```diff
@@ -2,32 +2,9 @@
 #define VARIABLES_H
 
 #include "z64.h"
-#include "libc64/os_malloc.h"
-#include "segment_symbols.h"
 
-struct MapData;
-
-extern OSPiHandle* gCartHandle;
-
-extern const char gBuildCreator[];
-extern const char gBuildDate[];
-extern const char gBuildMakeOption[];
-
-extern OSMesgQueue gPiMgrCmdQueue;
-extern OSViMode gViConfigMode;
-extern u8 gViConfigModeType;
-
-extern s16 gSpoilingItems[3];
-extern s16 gSpoilingItemReverts[3];
-
-// 4 16-colors palettes
-extern u64 gMojiFontTLUTs[4][4]; // original name: "moji_tlut"
-extern u64 gMojiFontTex[]; // original name: "font_ff"
-
-extern struct MapData* gMapData;
-extern u8 gBossMarkState;
-extern f32 gBossMarkScale;
-extern u32 D_8016139C;
-extern PauseMapMarksData* gLoadedPauseMarkDataTable;
+// TODO:
+// Plenty of files depend on this file to have access to z64.h, including assets.
```

Comment:
is this even true? I thought most assets either import no headers or import z64.h

Reply:
All scene/room files include variables.h for whatever reason. But I missed that they also import z64.h explicitly, so its probably not an issue here. I'll remove last couple words about assets.

Reply:
I think I found a simple fix. Just update global.h to include z64.h. You can also remove variables.h from global.h

Reply:
right, but you should be able to decouple variables.h from everything now by moving the `include "z64.h"` into global.h

Reply:
I don't really see the need for the bandaid fix for now when I'm going to properly fix it right after this is reviewed and merged. 

It would just shuffle things around and require all the bss fix.